lib/mailroom/imap/envelope.ex

```diff
@@ -48,7 +48,8 @@ defmodule Mailroom.IMAP.Envelope do
     bcc = parse_addresses(bcc)
 
     %__MODULE__{
-      date: date,
+      plane_date: date,
```

Comment:
Rather than `plane_date` (the English should probably be `plain_date`), how about `date_string`?

Reply:
done